### PR TITLE
Add yammer info to readme

### PR DESCRIPTION
**Pull request by fatboypunk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,27 +2,18 @@
...
-[this script]: https://github.com/thoughtbot/laptop
-
-After setting up, you can run the application using [Heroku Local]:
-
-    % heroku local
-
-[Heroku Local]: https://devcenter.heroku.com/articles/heroku-local
-
-## Guidelines
+We currently only support yammar to authenticate. To login on your local
```
Comment (dkhgh) [inline README.md]: YAMAR

Comment (fatboypunk) [inline README.md]: ?

Comment (dkhgh) [inline README.md]: You wrote yammar

Comment (fatboypunk) [inline README.md]: :dash:

Comment (dkhgh) [inline README.md]: what :ghost: this did not happen

**Pull request merged**